### PR TITLE
[Fixes] Fix target-path reference for Universal Package Publish

### PR DESCRIPTION
**Pull request by AlexanderSehr**
# Description

- Updated the reference to the module's deploy file to its parent folder

image

Downloaded artifact
image

| Pipeline |
| - |
| [Build Status](https://dev.azure.com/carml/CARML/_build/latest?definitionId=46&branchName=users%2Falsehr%2F2829_publish) |

# Type of Change

Please delete options that are not relevant.

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Update to documentation

### CODE REVIEW

File: modules/Microsoft.KeyVault/vaults/version.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
     "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
-    "version": "0.5"
+    "version": "0.6"
```
Comment (eriqua) [inline modules/Microsoft.KeyVault/vaults/version.json:3]: Is this bump intended to test the correct publishing or should it be reverted?

Comment (AlexanderSehr) [reply modules/Microsoft.KeyVault/vaults/version.json:3]: should be reverted. Good catch

Comment (AlexanderSehr) [reply modules/Microsoft.KeyVault/vaults/version.json]: Fixed

**Review by eriqua — APPROVED** [review]
🆗

**Pull request merged**